### DISCUSSION TITLE
Pizza Gücü

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#522] [Soru Cevap] [answered]
PizzaForces, Petya'nın en sevdiği pizzacı. Pizzalar üç boyuttadır: küçük pizzalar 6 dilim, orta pizzalar 8 dilim ve büyük pizzalar 10 dilim içerir. Küçük pizzaların pişirme süresi 15 dakika, orta pizzaların 20 dakika ve büyük pizzaların 25 dakikadır.

Petya'nın doğum günü bugün ve n arkadaşının geleceği için, at least n dilim pizza siparişi vermek istiyor. Siparişin pişirme süresi, sipariş edilen pizzaların toplam pişirme süresidir.

Göreviniz, en az n dilim içeren pizzalar pişirmek için gereken en az dakika sayısını belirlemektir. Örneğin:

- 12 arkadaş gelirse, toplam 12 dilim pizza sipariş etmelidir. İki küçük pizza siparişi verirse, 12 dilim eder ve toplam pişirme süresi 30 dakikadır.
- 15 arkadaş gelirse, toplam 15 dilim pizza sipariş etmelidir. Bir küçük ve bir büyük pizza siparişi verirse, 16 dilim eder ve toplam pişirme süresi 40 dakikadır.
- 300 arkadaş gelirse, 15 küçük pizza, 10 orta pizza ve 13 büyük pizza sipariş ederse, toplamda 300 dilim pizza eder ve toplam pişirme süresi 750 dakikadır.
- Sadece bir arkadaş gelirse, bir küçük pizza siparişi verirse, pişirme süresi 15 dakikadır.

**Input**
İlk satırda bir tam sayı t (1≤t≤10^4) — test sayısı vardır. Her test durumu, Petya'nın arkadaş sayısını belirten bir tam sayıyı (1≤n≤10^16) içerir.

**Output**
Her test durumu için, en az n dilim pizza pişirmek için gereken en az dakika sayısını yazdırın.

**Örnek**
Girdi:
```
6
12
15
300
1
9999999999999999
3
```
Çıktı:
```
30
40
750
15
25000000000000000
15
``` 

Bu şekilde PizzaForces'tan sipariş vererek arkadaşlarınıza yeterli dilim pizza sağlamanın en az süresini hesaplayabilirsiniz.

https://codeforces.com/problemset/problem/1555/A

**Comment by FurkanYildirim1350** [comment, marked as answer]
```
n=int(input("arkadaş sayısı:"))
a=[]
süre=0
yy=n//120
n=n%120
for b in range(0,12):
    for o in range(0,16):
        for k in range(0,20):
            if((b*10)+(o*8)+(k*6)==n):
                süre=(b*25)+(o*20)+(k*15)
                a.append(süre)
                a.sort()
if(n<=5):
    print("15")
else:
    print(a[0]+yy*300)

```

**Comment by Sevvalgocgun1** [comment]
```
import math

t = int(input())  # Test sayısını al
for _ in range(t):
    n = int(input())  # Dilim sayısını al
    min_time = math.ceil(n / 6) * 15  # 6'nın katına yuvarlayarak süreyi hesapla
    print(min_time)
```
